### ISSUE TITLE
Multi-line strings in apb.yml get corrupted during prepare

### ISSUE THREAD

**Issue by aweiteka** [fusor/ansible-playbook-bundle#84]
When the apb.yml spec file has a multi-line string, e.g....

```yaml
metadata:
  documentationUrl: https://access.redhat.com
  imageUrl: ansibleplaybookbundle/mongodb-apb
  dependencies: []
  displayName: MongoDB-Persistent (APB)
  longDescription: |-
    This template provides a standalone MongoDB server with a database
    created. The database is stored on persistent storage.  The database
    name, username, and password are chosen via parameters when provisioning
    this service.
```

...running apb prepare corrupts the file so it's no longer valid yaml, e.g.

```yaml
metadata:
  documentationUrl: https://access.redhat.com
  imageUrl: ansibleplaybookbundle/mongodb-apb
  dependencies: []
  displayName: MongoDB-Persistent (APB)
  longDescription: This template provides a standalone MongoDB server with a database
created. The database is stored on persistent storage.  The database
name, username, and password are chosen via parameters when provisioning
this service.
```

It's unexpected to have the apb.yml file change when running prepare. I made edits to it that are intentional. I understand this can be disabled with --skip-spec-update opt but why do we do it?

**Comment by dymurray** [comment]
@aweiteka We just changed the functionality of `apb prepare` to stop rewriting the spec file. This was initially a feature we thought would be nice to have for adding dependency information but it was causing too many issues such as this. You should no longer have to pass in the --skip-spec-update flag. I have confirmed that `apb prepare` now works without overwriting your spec.

**Comment by dymurray** [comment]
I will close this issue if you can confirm it is resolved.

**Comment by dymurray** [comment]
This is fixed in 0.2.1 on pypi.

**Comment by dymurray** [comment]
Confirmed this is fixed in the 0.2.1 rpm as well.

**Comment by aweiteka** [comment]
Thanks!
